anta/tests/routing/bgp.py

```diff
@@ -1226,3 +1226,88 @@ def test(self) -> None:
             self.result.is_success()
         else:
             self.result.is_failure(f"Following BGP peers are not configured or hold and keep-alive timers are not correct:\n{failures}")
+
+
+class VerifyBGPPeerUpdateErrors(AntaTest):
+    """Verifies if the IPv4 BGP peer's update errors counters should be zero.
+
+    Expected Results
+    ----------------
+    * Success: The test will pass if the BGP peer's update error counters are zero.
+    * Failure: The test will fail if the BGP peer's update error counters are non-zero.
+
+    Examples
+    --------
+    ```yaml
+    anta.tests.routing:
+      bgp:
+        - VerifyBGPPeerUpdateErrors:
+            bgp_peers:
+              - peer_address: 172.30.11.1
+                vrf: default
+                errors:
+                  - inUpdErrWithdraw
+    ```
+    """
+
+    name = "VerifyBGPPeerUpdateErrors"
+    description = "Verifies the update error counters of a BGP IPv4 peer."
+    categories: ClassVar[list[str]] = ["bgp"]
+    commands: ClassVar[list[AntaCommand | AntaTemplate]] = [AntaTemplate(template="show bgp neighbors {peer} vrf {vrf}")]
```

Comment:
Please add the revision number for the command. Thanks

Reply:
Added most current revision 3.